capstone-app/src/app/modules/history/wiki-media/wiki-media.component.ts

```diff
@@ -40,13 +40,13 @@ export class WikiMediaComponent implements OnChanges, OnInit {
 
   @Input() cityName!: string;
 
-  constructor(private cd: ChangeDetectorRef, private cityFetcher: SharedPlacesCityService, private wikiService: WikiResultsService) { }
+  constructor(private cityFetcher: SharedPlacesCityService, private wikiService: WikiResultsService, public zone: NgZone) { }
 
   ngOnInit(): void {
     this.cityFetcher.getCityName().subscribe(city => {
       this.cityName = city;
       if (city) {
-        this.getResults(this.cityName);
+        this.zone.run(() => this.getResults(this.cityName));
```

Comment:
I'm guessing why this was necessary is that it performs a subscribe inside of another subscribe block. Not necessary in this cl, but it might prevent other bugs to refactor so this does not occur.